tests/integration/cattletest/core/test_host_stats.py

```diff
@@ -54,8 +54,8 @@ def test_stats_container(admin_client, docker_context):
     stats_access = container.stats()
 
     assert stats_access.token.index('.') > 0
```

Comment:
@cjellick is this test just for native containers, Craig? And if its for both - regular, and native containers - does externalId get set to UUID for regular containers?